tests/cli/test_vehicle.py

```diff
@@ -55,6 +56,7 @@
         "GPS Latitude      48.1234567\n"
         "GPS Longitude     11.1234567\n"
         "GPS last updated  2020-02-18 17:58:38\n"
+        "Lock Status       locked"
```

Comment:
Missing new line (`\n`) in the expected output:
```suggestion
        "Lock Status       locked\n"
```